excludes/de.php

```diff
@@ -12,8 +12,108 @@
 
 return [
     'Administrator',
+    'Afghanistan',
+    'Andorra',
+    'Angola',
+    'Anguilla',
+    'April',
+    'Aruba',
+    'August',
+    'Bahamas',
+    'Bahrain',
+    'Barbados',
+    'Belarus',
+    'Belize',
+    'Benin',
+    'Bermuda',
+    'Bhutan',
+    'Botswana',
+    'Burkina Faso',
+    'Burundi',
+    'Chile',
+    'China',
     'Code',
+    'Costa Rica',
+    'Curaçao',
     'Dashboard',
+    'Details',
```

Comment:
`Details` > `Einzelheiten`?

Don't translate this word for sure?
Google Translate says there is a translation.

Reply:
Yes I am sure.
Details can be translated with details.
Check "More details" in Google Translate. It translates it with "Mehr Details".
In German we often use the word "Details" for "show me more information about an object"

Reply:
Got it. Just asked :)

I know that Google Translate doesn't translate exactly, but this word has a confirmation badge from people.

I think the confirmation is out of context.

![image](https://user-images.githubusercontent.com/10347617/115033376-0d199300-9ed3-11eb-9c0d-22d4db6c60bf.png)